File: python/permMissingElem.py
# you can use print for debugging purposes, e.g.
# print "this is a debug message"

def solution(A):
    # write your code in Python 2.7
    size = len(A) + 1
    sum_num = sum(A)
    return (((1 + size)*size)/2) - sum_num
    pass

File: cpp/permMissingElem.cpp
// you can use includes, for example:
// #include <algorithm>

// you can write to stdout for debugging purposes, e.g.
// cout << "this is a debug message" << endl;

#include <numeric>      // std::accumulate

int solution(vector<int> &A) {
    // write your code in C++11
    long long size = A.size() +1; 
    long long expectedSum = size * (1 + size) / 2;
    long long calculatedSum = std::accumulate(A.begin(), A.end(), 0);
    return expectedSum - calculatedSum;
}

